.pre-commit-config.yaml

```
@@ -15,6 +15,7 @@ repos:
   - id: requirements-txt-fixer
   - id: check-docstring-first
   - id: name-tests-test
+    args: [--django]
```

How about putting it on a dedicated line?
```suggestion
    args:
    - --django
```

---

Edited

---

Marking as resolved.